text/0000-rtsan-and-attributes.md

```diff
@@ -113,11 +113,11 @@ This process to enable a sanitizer has been completed by many of the other LLVM
 
 ### The addition two new outer attributes to rust - `#[nonblocking]` `#[blocking]`
 
-`#[nonblocking]` defines a scope as real-time constrained. During this scope, one cannot call any intercepted call (`malloc`, `socket` etc) or call any function marked `#[blocking]`.
+`#[nonblocking]` defines a scope as real-time constrained. During this scope, one cannot call any intercepted call (`malloc`, `socket` etc) or call any function marked `#[blocking]`. The rustc front-end will parse this attribute and add the LLVM attribute `llvm::Attribute::SanitizeRealtime`. This will leave most of the heavy lifting to the already-implemented LLVM instrumentation pass.
```

Comment:
Added a little implementation info here on how this will be done, same on line 118